src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+
+function calculateAverageAge(people) {
+  return people.reduce((acc, person) =>
+    acc + (person.died - person.born), 0) / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menFilteredAges = people
+    .filter(person => century
+      ? person.sex === 'm'
+        && Math.ceil(person.died / 100) === century
```

Comment:
The result of this calculation can be placed in a separate variable for better readability.